docs/release-notes/NuGet-5.6.md

```diff
@@ -0,0 +1,65 @@
+# 5.6 Release Notes
+
+[Full Changelog]("")
+
+[Issues List](https://app.zenhub.com/workspaces/nuget-client-team-55aec9a240305cf007585881/reports/release?release=5e3b2080c4b30708e48bf9f3)
+
+**DCR:**
+
+* Support pre-release packages with floating versions. Version="*-*" and similar. - [#912](https://github.com/NuGet/Home/issues/912)
+
+* add support to `NuGet.exe update` for -DependencyVersion parameter, like install command - [#7694](https://github.com/NuGet/Home/issues/7694)
+
+* Default verbosity should not report each project's noop restore. - [#8792](https://github.com/NuGet/Home/issues/8792)
+
+* Sort packages by ID in the Updates tab of the Package Manager UI - [#9278](https://github.com/NuGet/Home/issues/9278)
+
+* net5.0 tfm - initial support - [#9584](https://github.com/NuGet/Home/issues/9584)
+
+**Bug:**
+
+* dotnet nuget push - Missing value for option - [#4864](https://github.com/NuGet/Home/issues/4864)
+
+* `nuget push *.nupkg` fails with snupkg does not exist - [#8148](https://github.com/NuGet/Home/issues/8148)
+
+* Pack, and several other code paths, fails dependant on locale. Use RegexOptions.CultureInvariant - [#8246](https://github.com/NuGet/Home/issues/8246)
+
+* Perf: DG Spec for unloaded project scenarios should not be written in preview restores - [#8793](https://github.com/NuGet/Home/issues/8793)
+
+* Restore:  improve performance by caching solution dependency graph spec - [#9201](https://github.com/NuGet/Home/issues/9201)
+
+* NuGet UI doesn't work for sdk style projects after installing a  package with PMConsole - [#9203](https://github.com/NuGet/Home/issues/9203)
+
+* [Test Failure] Embedded icon can’t be shown in PM UI with local package feed - depending on / vs \ - [#9225](https://github.com/NuGet/Home/issues/9225)
+
+* NuGetVersion.TryParseStrict() should return false if it fails to parse - [#9255](https://github.com/NuGet/Home/issues/9255)
+
+* `nuget.exe push` help for `-source`, should be source name, not source uri - [#9265](https://github.com/NuGet/Home/issues/9265)
+
+* `dotnet nuget add package SourceUri`  creates bad default package source name - [#9277](https://github.com/NuGet/Home/issues/9277)
+
+* Screen reader doesn't announces "Searching..." message when switching tabs - [#9307](https://github.com/NuGet/Home/issues/9307)
+
+* [Test Failure] Focus-rect color in themes for PM UI tabs - [#9336](https://github.com/NuGet/Home/issues/9336)
+
+* Investigate Accessibility bugs reported by the Community - [#9393](https://github.com/NuGet/Home/issues/9393)

```

Comment:
This seems like a weird one to include. 

We need a better approach to our release note creation tbh :/

Reply:
we ended up editing that in the 2nd revision to:
* Several accessibility bugs reported by users - [#9393](https://github.com/NuGet/Home/issues/9393)

@donnie-msft  - did you end up fixing all those bugs as part of this issue?

Reply:
https://github.com/NuGet/Home/issues/9393#issuecomment-630527925

Reply:
There is nothing here to put into release notes, as there's no PR associated.

Reply:
ok. if something is marked as a bug. and associated with a release. it will be included in release notes. 
i marked it duplicate..since their was no fix, but several related issues spun out of this. (the release notes tool will filter it out automatically with that metadata) ... we will manually remove from this PR now. thx.